<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgGestionPersonas</class>
 <widget class="QWidget" name="dlgGestionPersonas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti√≥n de personas</string>
  </property>
  <widget class="QTableView" name="tvPersonas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>948</width>
     <height>588</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>948</width>
     <height>588</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckObispos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>84</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Obispos</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckAutores">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>84</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Autores</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>163</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filtro</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtFiltro">
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
